WapTenantPublicAPI.psm1

```diff
@@ -549,6 +549,192 @@ function GetWAPSubscriptionQuota {
     
 }
 
+function Get-WAPSubscriptionQuota{
```

Comment:
add space between function name and opening curly

Did you know there is already GetWAPSubscriptionQuota helper function? Could you investigate to what extent the logic is duplicate and if one can replace the other entirely or not?